### PR TITLE
SMART START guide

### LINKED ISSUE #8254: [Smart start] Integrate doc module
Add /smart-start using discourse module with existing [SmartStart category](https://discourse.ubuntu.com/c/smartstart)

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Add new discourse module with authentication

## QA

- Open the demo and navigate to `/docs`
- Might want to review the other `/docs` just in case, as there has been some renaming. i.e. `/server/docs`, `/tutorials`, `/ceph/docs`

## Issue / Card

Fixes #8254 #8255

### CODE REVIEW

**Comment by pmahnke** [comment]
I am getting a 

```
requests.exceptions.HTTPError
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://discourse.ubuntu.com/t/15433.json```

error

**Comment by carkod** [comment]
> I am getting a
> 
> ```
> requests.exceptions.HTTPError
> requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://discourse.ubuntu.com/t/15433.json```
> 
> error
> ```

Fixed. Can you take a look now?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Please update the content to have h1 followed by h2 for SEO. Comment in the code.

**Comment by carkod** [comment]
> Please update the content to have h1 followed by h2 for SEO. Comment in the code.

Done

File: templates/smart-start/document.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+{% extends "docs/base_docs.html" %}
+
+{% block title %} {{ document.title }} | Server documentation{% endblock %}
```
Comment (anthonydillon) [inline templates/smart-start/document.html:4]:
```suggestion
{% block title %} {{ document.title }} | Smart start{% endblock %}
```

**Review by anthonydillon — APPROVED** [review]
LGTM with one small change suggested.

**Pull request merged**